charts/kcp/templates/logical-cluster-admin-kubeconfig.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+stringData:
+  kubeconfig: |
+    apiVersion: v1
+    clusters:
+    - cluster:
+        certificate-authority: /etc/kcp/tls/server-client/ca.crt
+        server: https://kcp:6443
```

Comment:
does the need to come values `externalHostname`  field ?

Reply:
I don't think so - the logical cluster admin credentials used by controllers on the shard, so we don't want internal controller API traffic going via the external endpoint.  That said it looks like we actually [rewrite this URL](https://github.com/kcp-dev/kcp/blob/main/pkg/reconciler/tenancy/workspace/workspace_reconcile.go#L72) anyway.